### PR TITLE
SAR-8624 [PI-FE] Added Limited Partnership endpoint

### PR DESCRIPTION
**Pull request by refaque-ahmad**

### CODE REVIEW

File: app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  private val defaultPageConfig = PageConfig(
+    optServiceName = None,
+    deskProServiceId = "vrs",
+    signOutUrl = appConfig.vatRegFeedbackUrl
+  )
+
+  private val defaultJourneyConfig = JourneyConfig(
+    continueUrl = s"${appConfig.selfUrl}/identify-your-partnership/test-only/retrieve-journey",
+    pageConfig = defaultPageConfig,
+    ScottishPartnership
```
Comment (HarrNM) [inline app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala:48]: Entity type needs updating

File: app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  )
+
+  val show: Action[AnyContent] = Action.async {
+    implicit request =>
+      authorised() {
+        Future.successful(
+          Ok(view(defaultPageConfig, form(ScottishPartnership).fill(defaultJourneyConfig), routes.TestCreateLimitedPartnershipJourneyController.submit()))
```
Comment (HarrNM) [inline app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala:55]: Here as well

File: app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala (hunk 1 above)
Comment (refaque-ahmad) [reply app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala]: done

File: app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala (hunk 2 above)
Comment (refaque-ahmad) [reply app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/controllers/TestCreateLimitedPartnershipJourneyController.scala]: done

**Pull request merged**